docs/modules/ROOT/partials/cmos-health-checks.adoc

```diff
@@ -24,6 +24,18 @@ However, repeated crashes should be investigated as they may be caused by an und
 
 *Remediation*: Contact Couchbase Technical Support.
 
+[#CB90041]
+=== CPU Steal (CB90041)
+
+*Background*: Steal time is the percentage of time a virtual CPU waits for a real CPU while the hypervisor is servicing another virtual processor.
+In virtual environments, high CPU steal indicates that the virtual machines might be undersized or the hypervisor may be overcommitted.
+
+*Condition*: CPU steal rate is greater than 3%.
+
+*Remediation*: Increase resources available to virtual machine.
+
+*Further Reading*: https://support.couchbase.com/hc/en-us/articles/213960606-Virtualization-Considerations[Virtualisation Considerations]
```

Comment:
This link won't be accessible to people without a Support contract - if there's nothing better it's fine, but perhaps finding something on docs.couchbase.com?